src/views/penaltydetails.njk

```diff
@@ -0,0 +1,68 @@
+{% extends "layout.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/back-link/macro.njk"  import govukBackLink %}
+{% from "govuk/components/input/macro.njk"      import govukInput %}
+{% from "govuk/components/details/macro.njk"    import govukDetails %}
+{% from "govuk/components/button/macro.njk"     import govukButton %}
+
+
+{% block pageTitle %}
+  Tell us the penalty details
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        {{ govukErrorSummary
+          ({
```

Comment:
I suggest the following formatting for consistency:

<img width="491" alt="image" src="https://user-images.githubusercontent.com/1724691/74445138-cfa01c80-4e6d-11ea-8053-fcdcf1a9d9d7.png">